packages/shared/types/projects.ts

```diff
@@ -23,4 +23,10 @@ export type Project = {
   createdAt: number;
   updatedAt: number;
   size: number;
+  packageStatus?: {
+    [packageName: string]: {
+      isOutdated: boolean;
+      isUpdated?: boolean;
```

Comment:
Do we need both `isOutdated` and `isUpdated`?

Is it possible for both to be `true` or `false` at the same time?

Reply:
> Is it possible for both to be true or false at the same time?

nop, the flag: `isOutdated` works when the setting: `notify me` is turned on and the flag: `isUpdated` when the setting `auto-update` is turned on.

mmmm, I could use `isOutdated?: boolean` and show the notification message when the flag is !== `undefined`

Reply:
ah okay, so the `isOutdated` boolean always says whether the dependency is up-to-date or not, and the `isUpdated` is true only when we want to show the 'dependency have been updated'  notification. Maybe I would make the latter more literal, like `showUpdatedNotification` instead of `isUpdated`, otherwise it might not be clear which of the two properties is actually telling if the dependency is outdated or not.